packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -1160,6 +1154,9 @@ export function serializeRequestValue(
         return `${clientValue} as any`;
       }
     default:
+      if (clientValue === "constructorParam") {
```

Comment:
is this the only case? do we need to do this for other xxxParam?

Reply:
as offline discussed, this is the only case for now.